packages/core/src/App/Containers/RealAccountSignup/personal-details.jsx

```diff
@@ -149,6 +153,15 @@ const InputField = (props) => {
     );
 };
 
+const trimStrings = (values) => {
+    Object.keys(values)
+        .forEach(key => {
+            if (typeof values[key] === 'string' && values[key].startsWith(' ')) {
```

Comment:
```suggestion
            if (typeof values[key] === 'string' && (values[key].startsWith(' ') || values[key].charAt(values[key].length - 1) === ' ')) {
```

This makes it so that it removes trailing spaces too in case there is no leading space.